example/storybook/src/overview/Introduction/index.stories.mdx

```diff
@@ -9,16 +9,17 @@ import { Canvas, Meta, Story } from '@storybook/addon-docs';
 
 # Introduction
 
-Dank is a high-performing styling **CSS-in-JS** library for React and React Native that allows developers to create scalable and maintainable styles quickly and effectively.
+Dank is a high-performing **styling** library for React and React Native that allows developers to create **scalable and maintainable styles** quickly and effectively.
 
 ## Why we built dank.style?
 
-Dank is different from other CSS-in-JS libraries like Glamour, Emotion, Styled-components, and Vanilla-extract because it focuses on React Native while still being universal and high-performing on the web. Unlike other libraries, dank's styles are calculated during compile-time using a babel-plugin, which reduces the amount of processing needed during runtime. This means that with Server-Side Rendering (SSR) and Static Site Generation (SSG), pages created with dank can even run without JavaScript on the client, including media queries.
+Dank is different from other styling libraries like Glamour, Emotion, Styled-components, and Vanilla-extract because it focuses on React Native while still being universal and high-performing on the web. Unlike other libraries, dank's styles are calculated during compile-time **using a babel-plugin**, which reduces the amount of processing needed during runtime. This means that with Server-Side Rendering (SSR) and Static Site Generation (SSG), pages created with dank can even run without JavaScript on the client, including media queries.
 
-Dank originally started as part of NativeBase, a universal component library for both React and React Native. However, we realized that not everyone needs all the components from NativeBase, so we broke it down into two separate libraries:
+Dank originally started as part of NativeBase, a universal component library for both React and React Native. However, we realized that not everyone needs all the components from NativeBase, so we broke it down into three separate libraries:
```

Comment:
@amars29 `two` not `three`

Reply:
@amars29, dont change this part. Refer to the current doc and keep this part as it is!